src/sentry/apidocs/hooks.py

```diff
@@ -204,107 +204,128 @@ def custom_preprocessing_hook(endpoints: Any) -> Any:  # TODO: organize method,
-
-                        # There is an edge case where a body param might be
-                        # reference that we should ignore for now
-                        if "description" not in param_data and "$ref" not in param_data:
-                            raise SentryApiBuildError(
-                                f"""Body parameter '{body_param}' is missing a description for endpoint {endpoint_name}. You can either:
-                            1. Add a 'help_text' kwarg to the serializer field
-                            2. Remove the field if you're using an inline_serializer
-                            3. For a DRF serializer, you must explicitly exclude this field by decorating the request serializer with
-                            @extend_schema_serializer(exclude_fields=[{body_param}])."""
-                            )
-
-                    # Required params are stored in a list and not in the param itself
-                    required = set(schema.get("required", []))
-                    if required:
-                        # Explicitly sort body params by converting the dict to an ordered dict
-                        schema["properties"] = OrderedDict(
-                            sorted(
-                                schema["properties"].items(),
-                                key=lambda param: 0 if param[0] in required else 1,
-                            )
-                        )
-                except KeyError as e:
-                    raise SentryApiBuildError(
-                        f"Unable to parse body parameters due to KeyError {e} for endpoint {endpoint_name}. Please post in #discuss-apis to fix."
-                    )
+            try:
+                requestBody = method_info.get("requestBody")
+                if requestBody is not None:
+                    _validate_request_body(requestBody, schema_components, endpoint_name)
+            except KeyError as e:
+                raise SentryApiBuildError(
+                    f"Unable to parse body parameters due to KeyError {e} for endpoint {endpoint_name}. Please post in #discuss-apis to fix."
+                )
     return result
 
 
-def _check_tag(path: str, method_info: Mapping[str, Any]) -> None:
+def _check_tag(method_info: Mapping[str, Any], endpoint_name: str) -> None:
     if method_info.get("tags") is None:
         raise SentryApiBuildError(
-            f"Please add a single tag to {path}. The list of tags is defined at OPENAPI_TAGS in src/sentry/apidocs/build.py "
+            f"Please add a single tag to {endpoint_name}. The list of tags is defined at OPENAPI_TAGS in src/sentry/apidocs/build.py "
         )
 
     num_of_tags = len(method_info["tags"])
 
     if num_of_tags > 1:
         raise SentryApiBuildError(
-            f"Please add only a single tag to {path}. Right now there are {num_of_tags}."
+            f"Please add only a single tag to {endpoint_name}. Right now there are {num_of_tags}."
```

Comment:
It's much easier to find the incorrect endpoint based on it's name rather than URL path